### DISCUSSION TITLE
Add :query-shell-command

### DISCUSSION THREAD

**Discussion by Sntx626** [helix-editor/helix#8944] [Ideas] [answered]
I'd like to be able to run a shell command and have helix run the returned text of the shell command as an internal command.

The name for the command is just a suggestions.

I.e.:

> ### helix
> `:query-shell-command /usr/bin/my-custom-script.sh`
>
> ### my-custom-script.sh
> ```bash
> #!/bin/bash
> DATE=$(date +'%Y-%m-%d')
> FILENAME=~/Documents/journal/$DATE.md 
>
> cat > $FILENAME <<EOL
> # ${DATE} journal entry
>
> 
> EOL
>
> echo ":open $FILENAME"
> ```

**Comment by the-mikedavis** [comment, marked as answer]
See https://github.com/helix-editor/helix/pull/8636
